head/src/main/kotlin/io/qalipsis/core/head/report/ReportsPurgeManager.kt

```diff
@@ -40,23 +46,31 @@ import java.time.Instant
     Requires(beans = [ReportRecordsTTLConfiguration::class, ReportTaskRepository::class, ReportFileRepository::class]),
     Requires(notEnv = [AUTOSTART])
 )
+@PropertySource(
```

Comment:
The properties should not be set that way, but as default values in the application.yml